File: library/src/main/java/com/kimcy929/iconpakagereader/customview/WrapContentGirdLayoutManager.java
package com.kimcy929.iconpakagereader.customview;

import android.content.Context;

import androidx.recyclerview.widget.GridLayoutManager;
import androidx.recyclerview.widget.RecyclerView;

public class WrapContentGirdLayoutManager extends GridLayoutManager {

    public WrapContentGirdLayoutManager(Context context, int spanCount) {
        super(context, spanCount);
    }

    @Override
    public void onLayoutChildren(RecyclerView.Recycler recycler, RecyclerView.State state) {
        try {
            super.onLayoutChildren(recycler, state);
        } catch (IndexOutOfBoundsException ignored) {
        }
    }
}

File: library/src/main/java/com/kimcy929/iconpakagereader/customview/WrapContentGirdLayoutManager.kt
package com.kimcy929.iconpakagereader.customview

import android.content.Context
import androidx.recyclerview.widget.GridLayoutManager
import androidx.recyclerview.widget.RecyclerView

class WrapContentGirdLayoutManager(context: Context, spanCount: Int) : GridLayoutManager(context, spanCount) {

    override fun onLayoutChildren(recycler: RecyclerView.Recycler?, state: RecyclerView.State) {
        try {
            super.onLayoutChildren(recycler, state)
        } catch (ignored: IndexOutOfBoundsException) {
        }

    }
}